ide/che-core-ide-api/pom.xml

```diff
@@ -18,6 +18,7 @@
         <groupId>org.eclipse.che.core</groupId>
         <version>4.4.0-RC1-SNAPSHOT</version>
     </parent>
+    <groupId>org.eclipse.che.core</groupId>
```

Comment:
(could be done in a next commit)
to be sure groupId are aligned with the location of modules